geoh5py/objects/surveys/magnetotellurics.py

```diff
@@ -1,229 +0,0 @@
-#  Copyright (c) 2022 Mira Geoscience Ltd.
```

Comment:
**thought:** not really deleted but moved. We are losing the file history here.
Might have to do some interactive rebase to isolate the move operation, before editing code, so that git keeps track of the history